kivymd/uix/snackbar.py

```diff
@@ -309,7 +316,9 @@ class BaseSnackbar(MDCard):
     All Snackbars will be made off of this `BaseSnackbar`. `BaseSnackbar` will always try to fill the remainder of the screen with your Snackbar.
     To make your Snackbar dynamic and symetric with snackbar_x. Set size_hint_x like below:
 
-    `size_hint_z = (Window.width - (snackbar_x * 2)) / Window.width`
+    `size_hint_z = (
+        Window.width - (snackbar_x * 2)
+    ) / Window.width`
```

Comment:
Use `.. code-block:: python` block to insert code in docstrings - https://github.com/kivymd/KivyMD/blob/master/kivymd/uix/datatables.py#L673

Reply:
Fixed.